README.md

```diff
@@ -11,6 +11,15 @@ in modern frontend frameworks like Vue or React.
 
 With `django-components` you can support Django projects small and large without leaving the Django ecosystem.
 
+## Sponsors
+
+<p align="center"><strong>Platinum Sponsors</strong></p>
+<p align="center">
+  <a href="https://www.ohne-makler.net/" target="_blank"><img
```

Comment:
Add the tagline on hover-over + add identifier so they can potentially track traffic from this project.

```suggestion
  <a href="https://www.ohne-makler.net/?ref=django-components" target="_blank" title="Ohne-makler: Sell ​​and rent real estate without an agent"><img
```